tlslite/utils/codec.py

```diff
@@ -324,6 +324,12 @@ def getFixBytes(self, lengthBytes):
         self.index += lengthBytes
         return bytes
 
+    def skip_bytes(self, length):
+        """Move the internal pointer ahead length bytes."""
+        if self.index + length > len(self.bytes):
+            raise DecodeError("Read past end of buffer")
```

Comment:
Maybe different exception message like "Skipped past the end of buffer"?

Reply:
this method is used only to get to the point we want to read, so while "skipped past..." would make more sense on the level of the Parser class, for the classes higher up the stack I think "Read past end of buffer" is more understandable